pkg/ocm/ocm.go

```diff
@@ -93,6 +96,18 @@ func ZeroFields(obj runtime.Object) runtime.Object {
 	return zeroed
 }
 
+func GetOCMClientset(kubeconfig *rest.Config) (*clientset.Clientset, error) {
+	clientset, err := clientset.NewForConfig(kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+	return clientset, nil
+}
```

Comment:
you could write this function as:
`return clientset.NewForKubeconfig(kubeconfig)`

considering you agree with the above statement, I'm not sure why you need this function, you can just put this line in the controller itself instead of calling GetOCMClientSet helper function which wraps a single line.

Reply:
Done